Kernel/Bus/VirtIO/RNG.cpp

```diff
@@ -4,28 +4,23 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <Kernel/Bus/VirtIO/Initializer.h>
 #include <Kernel/Bus/VirtIO/RNG.h>
 #include <Kernel/Sections.h>
 
 namespace Kernel::VirtIO {
 
-UNMAP_AFTER_INIT RefPtr<RNG> RNG::try_create(PCI::Address address)
+UNMAP_AFTER_INIT Result<NonnullRefPtr<RNG>, InitializationResult> RNG::try_create(PCI::Address address)
```

Comment:
This is very weird, we have a Result object which is either a NonnullRefPtr or another result object? Cant we just make `InitializationResult` be a simple enum?